### DISCUSSION TITLE
Hugging Face API-Key Security

### DISCUSSION THREAD

**Discussion by kayx23** [GaiZhenbiao/ChuanhuChatGPT#550] [Q&A]
Wondering if there is any security concern around passing the API key to hugging face public app? Googled a bit but didn't find many useful information... Any comments or links on this topic welcomed!

**Comment by GaiZhenbiao** [comment]
This is a open source project, you can look into the source if you have security concerns :)

**Comment by Keldos-Li** [comment]
<img width="1479" alt="image" src="https://user-images.githubusercontent.com/23137268/230061418-05d29dcd-59f4-408b-a8e9-e733067c575c.png">
我不清楚现在是不是也用这个办法

**Reply by GaiZhenbiao** [reply]
是的，是这样的
